### DISCUSSION TITLE
mysqldump: Got error: 1449: The user specified as a definer ('admin'@'%') does not exist when using LOCK TABLES

### DISCUSSION THREAD

**Discussion by rajeshisnepali** [spatie/laravel-backup#1620] [Q&A]
Getting this error 
`
Backup failed because: The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Got error: 1449: The user specified as a definer ('admin'@'%') does not exist when using LOCK TABLES
`

**Comment by parallels999** [comment]
[**Error 1449:** The user specified as a definer](https://www.google.com/search?q=mysql+1449+The+user+specified+as+a+definer+(%27admin%27%40%27%25%27)+does+not+exist+when+using+LOCK+TABLES&oq=mysql+1449+The+user+specified+as+a+definer+(%27admin%27%40%27%25%27)+does+not+exist+when+using+LOCK+TABLES)
